packages/dds/tree/src/feature-libraries/chunked-forest/codec/chunkEncodingGeneric.ts

```diff
@@ -43,13 +43,13 @@ export type BufferFormat<TEncodedShape> = (
 )[];
 
 /**
- * Replace shapes and identifiers in buffer and any nested arrays.
+ * Encodes shapes and identifiers in buffer and any nested arrays.
```

Comment:
I think this rename is misleading.

Most "Encode" functions write something into the provided buffer.

This code does not do that.

It instead takes an existing buffer, into which content has already been encoded, then searches it for shapes and identifiers, and modifies how they are encoded in it.

Perhaps this rename isn't too bad, but we really need to emphasize the distinction in how we phrase the docs:

Maybe:

```suggestion
 * Takes already encoded content in `buffer` and deeply searches, replacing each `IdentifierToken` and `Shape` with references to their encodings in the returned `EncodedFieldBatchGeneric`.
```

Reply:
Renamed to `updateShapesAndIdentifiersEncoding` which may reflect what's happening better.